### ISSUE TITLE
Cannot replicate results with tensorflow on colab but can on jupyterlab

### ISSUE THREAD

**Issue by fickas** [googlecolab/colabtools#3726]
**Describe the current behavior**

I am setting both numpy and tensorflow seeds to a constant. Yet I get different results every time I train a sequential neural net. When I run the notebook on my laptop on jupyterlab, the problem does not occur - I get replicable results.

I have GPUs turned off on colab.

**Describe the expected behavior**

I expect to get the same results with same seed. Without this, it is impossible to do tuning.

**What web browser you are using**

Chrome

**Additional context**

[Link to runnable notebook](https://colab.research.google.com/drive/1wxa5PCf7kQhTKrDFlbehq-k8MGAQpwkK?usp=sharing)

**Comment by mayankmalik-colab** [comment]
I am just curious if you are using the same TF ( and perhaps other packages) version on both the platforms.?

**Comment by fickas** [comment]
Good point. I was not using same versions. When I updated to colab version on jupyterlab, I now get same problem. What I cannot get my head around is how to do tuning now. If I run grid-search or something fancier, it seems semi-useless. I'll get different "best parameters" on each run. This is exactly what is happening in my class when I ask students to tune their tensorflow models. I must be missing something. I would expect a big hue and cry when tensorflow updated to version that no longer is replicable. Don't see any complaints.

**Comment by cperry-goog** [comment]
So sorry - this doesn't look like a Colab issue - you'd want to file a but on TensorFlow
